cli/src/commands/logs.ts

```diff
@@ -48,10 +55,27 @@ $ ${Command.cliBinName} logs --since 2m --follow`,
   public static strict = true;
 
   public async run() {
+    const { since, follow, limit } = this.parse(Logs).flags;
```

Comment:
Why change the original destructuring syntax?  (I don't care one way or the other particularly, just seems strange to switch).

Reply:
I suspect for readability. I, too, like Roey find the new format more readable.